x-pack/plugins/security_solution/common/api/detection_engine/model/rule_schema/rule_schemas.ts

```diff
@@ -219,7 +219,7 @@ export const KqlQueryLanguage = t.keyof({ kuery: null, lucene: null });
 export type EqlQueryLanguage = t.TypeOf<typeof EqlQueryLanguage>;
 export const EqlQueryLanguage = t.literal('eql');
 
-const eqlSchema = buildRuleSchemas({
+export const eqlSchema = buildRuleSchemas({
```

Comment:
Why do we export types here and not reuse EqlRuleCreateProps for example?

Reply:
As I understand `EqlRuleCreateProps` describes common rule properties + EQL rule specific properties. Here I needed a way to get only EQL rule specific properties. But anyways I'm going to refactor this a little after FF and remove these exports.